Stripe/STPPaymentCardTextField.m

```diff
@@ -1340,6 +1340,10 @@ - (void)formTextFieldTextDidChange:(STPFormTextField *)formTextField {
                         [self updateCVCPlaceholder];
                         self.cvcField.validText = [self.viewModel validationStateForCVC] != STPCardValidationStateInvalid;
                         formTextField.validText = state != STPCardValidationStateInvalid;
+                        if (state == STPCardValidationStateValid) {
+                            // log that user entered full complete PAN before we got a network response
+                            [[STPAnalyticsClient sharedClient] logUserEnteredCompletePANBeforeMetadataLoadedWithConfiguration:[STPPaymentConfiguration sharedConfiguration]];
```

Comment:
Wouldn't this log on paste?

Reply:
yeah. I think that's a legitimate log. There might be an optimization to split out cases